lib/livebook_web/live/session_live/cell_component.ex

```diff
@@ -382,16 +385,22 @@ defmodule LivebookWeb.SessionLive.CellComponent do
       text: text,
       circle_class: circle_class,
       animated_circle_class: Keyword.get(opts, :animated_circle_class),
-      change_indicator: Keyword.get(opts, :change_indicator, false)
+      change_indicator: Keyword.get(opts, :change_indicator, false),
+      evaluation_time: Keyword.get(opts, :evaluation_time, 0)
     }
 
     ~L"""
     <div class="flex items-center space-x-1">
-      <div class="flex text-xs text-gray-400">
+      <div class="flex text-xs text-gray-400 space-x-1">
         <%= @text %>
         <%= if @change_indicator do %>
           <span data-element="change-indicator">*</span>
         <% end %>
+        <%= if @evaluation_time > 100 do %>
+          <span><%= @evaluation_time/1000 %>s</span>
+        <%= else %>
+          <span><%= @evaluation_time %>ms</span>
+        <% end %>
```

Comment:
Could you make this a tooltip? You can wrap the outermost div in a tooltip like this:

```eex
<span class="tooltip right" aria-label="<%= evaluated_label(@evaluation_time) %>">
  ...
</span>
```

And the tooltip can say something like "Took 0.2s".

Reply:
<img width="260" alt="Screen Shot 2021-06-18 at 9 24 46 AM" src="https://user-images.githubusercontent.com/252122/122567759-07494680-d017-11eb-890a-9e9a237c20bc.png">

Reply:
Perhaps add a "distant" class. If it's too distant we can figure out a different class too.

Reply:
We may not have space on the right, so maybe top or bottom is a better fit than distant?

Reply:
<img width="142" alt="Screen Shot 2021-06-18 at 9 35 47 AM" src="https://user-images.githubusercontent.com/252122/122569275-930fa280-d018-11eb-995a-47da70f34b09.png">
<img width="168" alt="Screen Shot 2021-06-18 at 9 36 34 AM" src="https://user-images.githubusercontent.com/252122/122569381-ade21700-d018-11eb-9c39-3ae166466770.png">

The CSS issues are layout related

Reply:
Top and left wouldn't be visible, we can try bottom though.

Reply:
<img width="193" alt="Screen Shot 2021-06-18 at 9 39 26 AM" src="https://user-images.githubusercontent.com/252122/122569780-14ffcb80-d019-11eb-8927-dc284cca5afa.png">
Yes, works better

Reply:
Looks good to me!